builder/llpcBuilder.cpp

```diff
@@ -534,3 +534,21 @@ Constant* Builder::Get180OverPi(
     return GetFpConstant(pTy, APFloat(APFloat::IEEEdouble(), APInt(64, 0x404CA5DC20000000)));
 }
 
+// =====================================================================================================================
+// Create a call to the specified intrinsic with one operand, mangled on its type.
+// This is an override of the same method in IRBuilder<>; the difference is that this one sets fast math
+// flags from the Builder if none are specified by pFmfSource.
+CallInst* Builder::CreateUnaryIntrinsic(
+    Intrinsic::ID id,           // Intrinsic ID
+    Value*        pValue,       // [in] Input value
+    Instruction*  pFmfSource,   // [in] Instruction to copy FMF from; nullptr to use Builder's FMF
```

Comment:
pFmfSource -> pFastMathFlags
FMF used in LLVM is somewhat unclear. Give it a meaningful naming here

Reply:
I gave it the same name as in the IRBuilder method it overrides, but translated into pal style. If you want to expand the "fmf", we should also keep the "source", as this argument is not the FMF, it is a pointer to the instruction that we want it to get the FMF from. So we end up with the somewhat verbose "pFastMathFlagsSource". Is that ok?

Reply:
OK. If it is a overriding function, then we keep them consistent. Ignore my comments. Just expand the comments // Instruction to copy FastMathFlags from